charts/open-notificaties/templates/configmap.yaml

```diff
@@ -24,6 +24,15 @@ data:
   {{- end }}
   IS_HTTPS: {{ if .Values.settings.isHttps }}"True"{{ else }}"False"{{ end }}
   RABBITMQ_HOST: {{ .Values.settings.messageBroker.host }}
+  {{- if not eq .Values.settings.autoRetry.maxRetries nil }}
+  NOTIFICATION_DELIVERY_MAX_RETRIES: "{{ .Values.settings.autoRetry.maxRetries }}"
+  {{- end }}
```

Comment:
Go templates are confusing... couldn't find from the docs how you actually check that something is `null`/`nil` so this is my best guess.

Basically, end-users can set this to `0` to opt out of auto-retry behaviour, so a simple boolean check is not suitable.